### DISCUSSION TITLE
Synapps T° non visible en Invité

### DISCUSSION THREAD

**Discussion by StephanBerio** [witsa/synapps#714] [General]
Sur le Synapps FFCAM Pelvoux joint, en log Invité FFCAM_Invite1 mdp FFCAM_Invite1 Autorisation Invité
Certaines températures ne s’affichent pas sur le Synapps Chaufferie (exemple Départ, Bouclage, Ballon ECS,..)
Alors que d’autres s’affichent bien exemple Plancher Rdc/Etages
Quand on passe simplement l’invité 1 en Exploitant, alors les températures s’affichent toutes… (sans changer les groupes)
[template_Pelvoux_FFCAM.zip](https://github.com/witsa/synapps/files/15160042/template_Pelvoux_FFCAM.zip)

**Comment by Raphwit** [comment]
Bonjour, premièrement, je ne vous conseille de ne pas partager des mots de passe sur GitHub, même temporaire.

Concernant votre soucis, effectivement en mode invité les ressources situées dans l'extenbus et l'embase UC ne sont pas accessibles, il n'y a rien à faire côté Synapps et c'est une limitation côté REDY.

**Comment by StephanBerio** [comment]
sur l'embase j'ai une température au Plug05 version V2.0.0 "T° Départ Plancher Rdc/Etage" visible en invité
en Extension P9 Armoire Chaufferie Plug08 version V2.0.0 une température "T° Bouclage ECS" non visible en invité
ça veut dire que sur l'Embase ça marche mais pas sur les Extenbus?

**Reply by Raphwit** [reply]
Oui c'est exact, l'extenbus ne permet pas aux invités d'accéder à des ressources, je note pour faire remonter l'anomalie.

Pour apporter un peu plus de précision sur ce cas, je ne sais pas si vous faites de traitement sur cette température dans le REDY mais il est préconisé d'exposer le résultat de ce traitement dans une autre ressource qui ne se situe pas dans l'extenbus.
Egalement, il est possible d'exposer l'état de cette ressources depuis une autre qui ne se situe également pas dans l'extenbus.

**Comment by WITNicolas** [comment]
Ce que tu peux faire, c'est mettre un compte EXPLOITANT et le mettre seulement en visu dans les Groupes de l'utilisateur.
Donc même avec un niveau EXPLOITANT il n'y aura pas de commande.

**Comment by StephanBerio** [comment]
Ok merci Nicolas
Pour les unités en liste reflet t'as pu regarder?

-----Message d'origine-----
De : WITNicolas ***@***.***> 
Envoyé : mardi 30 avril 2024 13:51
À : witsa/synapps ***@***.***>
Cc : StephanBerio ***@***.***>; Author ***@***.***>
Objet : Re: [witsa/synapps] Synapps T° non visible en Invité (Discussion #714)

Ce que tu peux faire, c'est mettre un compte EXPLOITANT et le mettre seulement en visu dans les Groupes de l'utilisateur.
Donc même avec un niveau EXPLOITANT il n'y aura pas de commande.

-- 
Reply to this email directly or view it on GitHub:
https://github.com/witsa/synapps/discussions/714#discussioncomment-9274536
